### DISCUSSION TITLE
Project 'aa-bb-cc-hello-pushnotifications' contains extension with invalid name (type).

### DISCUSSION THREAD

**Discussion by sshelake25** [nx-dotnet/nx-dotnet#903] [Q&A]
I have dotnet project which has project.json file like this. Whenever I build project I am getting warning. I am not sure how to resolve this. Any help really appreciated. 

{
	"name": "aa-bb-cc-hello-pushnotifications",
	"$schema": "../../node_modules/nx/schemas/project-schema.json",
	"projectType": "library",
	"sourceRoot": "libs/aa-bb-cc-hello-pushnotifications",
	"targets": {
		"build": {
			"executor": "@nx-dotnet/core:build",
			"outputs": [
				"{workspaceRoot}/dist/libs/shared-backend/Common/aa-bb-cc-hello-pushnotifications",
				"{workspaceRoot}/dist/intermediates/libs/shared-backend/Common/aa-bb-cc-hello-pushnotifications"
			],
			"options": {
				"configuration": "Debug",
				"noDependencies": true
			},
			"configurations": {
				"production": {
					"configuration": "Release"
				}
			}
		},
		"lint": {
			"executor": "@nx-dotnet/core:format"
		}
	},
	"tags": ["type:lib", "type:backend", "domain:common"]
}

**Comment by AgentEnder** [comment]
IIRC this was a bug we fixed in core nx
